test/main.go

```diff
@@ -335,3 +336,28 @@ func dartTests(ctx context.Context, client *dagger.Client, t *testCase) error {
 
 	return err
 }
+
+// csharpTests runs the csharp integration test suite against a container running Flipt.
+func csharpTests(ctx context.Context, client *dagger.Client, t *testCase) error {
+	platform := "linux/amd64"
+	if architecture == "arm64" {
+		platform = "linux/arm64"
+	}
```

Comment:
we should prob do this for all the tests so that dagger pulls the right Docker image dependent on the runner platform